src/Console/CommandCollection.php

```diff
@@ -203,6 +203,9 @@ protected function resolveNames(array $input): array
             }
         }
 
+        // This entry is created for the `Command` class, which we don't need.
+        unset($out['']);
```

Comment:
Could we exclude the base class in CommandScanner so it doesn't show up in other places as well?

Reply:
Done